staker/block_validator.go

```diff
@@ -174,7 +175,14 @@ func (c *BlockValidatorConfig) Validate() error {
 }
 
 type BlockValidatorDangerousConfig struct {
-	ResetBlockValidation bool `koanf:"reset-block-validation"`
+	ResetBlockValidation bool               `koanf:"reset-block-validation"`
+	Revalidation         RevalidationConfig `koanf:"revalidation"`
+}
+
+type RevalidationConfig struct {
+	StartBlock uint64 `koanf:"start-block"`
+	EndBlock   uint64 `koanf:"end-block"`
+	ThenQuit   bool   `koanf:"then-quit"`
```

Comment:
nitpick: I would change this to something like `QuitAfterRevalidation`

Reply:
Done.